pytorch_lightning/accelerators/tpu.py

```diff
@@ -43,6 +44,9 @@ def get_device_stats(self, device: Union[str, torch.device]) -> Dict[str, Any]:
         }
         return device_stats
 
+    def teardown(self) -> None:
+        os.environ.pop("PT_XLA_DEBUG", None)
```

Comment:
a teardown should not remove environment variables that it doesn't add itself.
This is done in the strategy, let's keep it there?